runtime/onert/backend/train/KernelGenerator.h

```diff
@@ -50,9 +50,11 @@ class KernelGenerator : public backend::train::KernelGeneratorBase
   void visit(const ir::train::operation::ElementwiseActivation &) override;
   void visit(const ir::train::operation::FullyConnected &) override;
   void visit(const ir::train::operation::Loss &) override;
+  void visit(const ir::train::operation::Pool2D &) override;
   void visit(const ir::train::operation::Reshape &node) override;
 
 private:
+  const ir::Operands &_ctx;
```

Comment:
`Operands _ctx` is necessary to get the maxpool in/out tensor shape. Shape is used to caculated maxpool-padding.